src/modules/Dashboard/components/ActivityUnassignDrawer/ActivityUnassignDrawer.schema.ts

```diff
@@ -0,0 +1,8 @@
+import * as yup from 'yup';
+
+export const useActivityUnassignFormSchema = () =>
+  yup
+    .object({
+      selectedAssignments: yup.array().min(1).required(),
+    })
+    .required();
```

Comment:
Yup, totes! I was just following the same pattern used elsewhere that react-hook-form is used, but this can totally be static and probably doesn't even need to be in its own file.